sql/src/main/java/io/crate/analyze/AlterTableAnalyzer.java

```diff
@@ -94,6 +89,8 @@ private static void maybeRaiseBlockedException(DocTableInfo tableInfo, Settings
              tableSettings.get(IndexMetaData.SETTING_READ_ONLY) == null)) {
 
             Operation.blockedRaiseException(tableInfo, Operation.ALTER);
+        } else {
```

Comment:
afaik this is not needed, its already checked, also pls rename or move where it is used